src/noLiteralsRule.js

```diff
@@ -0,0 +1,16 @@
+module.exports =  {
+      create: noLiteralsRule()
+}
+
+function noLiteralsRule(){

```

Comment:
does not care for the allowed strings in it and describe.
The following code should pass:

```
import {} from "jest"
import { MakeScheduleSelectionsService } from "com.kodekonveyor.smarthomecare/task/schedule/MakeScheduleSelectionsService"
import { ScheduleEntityTestData } from "../../entities/ScheduleEntityTestData"

describe("MakeScheduleSelectionsService", () => {
    const result = MakeScheduleSelectionsService([ScheduleEntityTestData.get()])

    it("empty input, empty output", () => {
        expect(MakeScheduleSelectionsService([])).toEqual([])
    })

    it("value is the id as string", ()=> {
        expect(result[0].value).toEqual(ScheduleEntityTestData.ID.toString())
    })

    it("label is the name", ()=> {
        expect(result[0].label).toEqual(ScheduleEntityTestData.NAME)
    })

})

```

Reply:
Now the logic works. Please make testcases as well.